explorerlib/configBuilder.go

```diff
@@ -26,12 +26,6 @@ func (builder *botConfigurationBuilder) WithLanguages(languages []string) *botCo
 	return builder
 }
 
-func (builder *botConfigurationBuilder) AllAnonymous() *botConfigurationBuilder {
-	builder.config.AllowAnonymous = true
```

Comment:
@erocheleau that prevented the build to be successful.
The Config object in uabot repo doesn't have the AllowAnonymous property anymore.

Reply:
ah, yes, change it so it sets the `AnonymousThreshold     float64` to 1 instead. And there is no longer a need to `allowAnonymous`.

Reply:
Just remove the line `builder.config.AllowAnonymous = true` and that should be it.